docs/data/material/getting-started/overview/overview.md

```diff
@@ -0,0 +1,26 @@
+# Material UI - Overview
+
+<p class="description">Material UI is library of React UI components that implements Google's Material Design.</p>
+
+## Introduction
+
+Material UI is an open-source React component library that implements Google's [Material Design](https://material.io/).
```

Comment:
```suggestion
Material UI is an open-source React component library that implements version 2 of Google's [Material Design](https://material.io/).
```

Reply:
We take the risk to forget to update it, with Material UI v6 which would cover Material Design v3 if everything goes smoothly.

If we really want to mention a version, then I think that we should rephrase it, in the current wording, it could sounds like v2 is our ultimate goal. Instead, we could say something around "currently v2". e.g.

```suggestion
Material UI is an open-source React component library that implements Google's [Material Design](https://material.io/).
It currently only supports Material Design v2, see [issue #12134](foo/bar) for the support of Material Design v3.
```

Reply:
I'm confused about what's the intended direction regarding Material's v3, it doesn't seem to exist alignment in this matter — see https://github.com/mui/material-ui/pull/32643#discussion_r867355637

Reply:
Alternatively: we could always just leave this out completely if we aren't prepared to say yes or no to v3 adoption. 😅